rebar.lock

```diff
@@ -1,78 +0,0 @@
-{"1.1.0",
```

Comment:
The `rebar.lock` is removed, maybe make sense add `rebar.lock` into `.gitignore`?

Reply:
removing the .lock file was a mistake, should only be updated.